[*] running method core_negotiate_tlv_encryption
[*] Negotiating TLV encryption
[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100dd2e77026312aab4ad2df2b5b0496c2fad757e916056156f94abdad0703b2f22f4adb36791b8276400ab7186cd88056c642ebd9519ad45a301aa74bce5af2fc4df2a01b075375041cb9a81180e10d56e3822f049dfd29df63937aa1e7b0ce8832db4f4297d111d2fa1621de794d5d3d573a7d86a5c3bdf79e01f938d411f03712378b08c9f31c2d405700ec58a9266fd67f38d7c06fc8c5825b395c4410d182f224f5e92afdd195eea88888f0039f5c9bb286dd01e6d7c8007fb0ac3eeeb640b6221f7df35122394d0cfeb4a78e3e0971620b1df12bd13bca71aca1962b782a10a5a60e1588aa372c6ffd65f76bd8914bd3f4d215fc48a21ff46f6662c3a93690203010001
[*] AES key: 0x4c9ece8b0df7e9fa6ccee6f4183495e3625e515bccddc771c946f5136ba19e88
[*] Encrypted AES key: 0xb26232761f47b5597976ed8a95a6d801bd1e0f6d75595eff30152b56591b06ac8ed7b07fbdb86d98e922da5a19f6f083e8761b1327e05495bfbb51e4af533ca2669caa682757a27dcf14cfeabdc40611fdbf29b432599a7f0d9e184827a700e99899958dcc44bc55f8b7afa3cb5fced626736520b5e3bf15b1afa07f85b67feb9375c44bff7276e44c37cd487ab76e536b91b7e9f7319598ec261b220034305ad0b90f8848c9cd21f8c3d9ea715d6e2b921e2b05be2233f862617e3969b21c17583e43e29a80fc012557fed6bc84c75ad1daef76fb758e2c9b060371493665b4f800e7080a7309b3cdb1b206f4999a3ab4e13e2e4da91ae48af9f66337974ccc
[*] TLV encryption sorted
[*] sending response packet
[*] running method core_set_session_guid
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_loadlib
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getuid
[*] sending response packet
[*] running method stdapi_sys_config_sysinfo
[*] sending response packet
[*] running method core_set_uuid
[*] sending response packet
[*] running method stdapi_net_config_get_interfaces
[*] sending response packet
[*] running method stdapi_net_config_get_routes
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_separator
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-kmhyozfa'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 1 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 2 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 3 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 3
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 4 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 4
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 5 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 5
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 6 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 6
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 7 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 7
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 8 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 8
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 9 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 9
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 10 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 10
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 11 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 11
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 12 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 12
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method stdapi_fs_file_move
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 13 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 13
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 14 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 14
[*] sending response packet
[*] running method stdapi_fs_ls
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo TKJKNEPQ']
[*] added process id: 3433, handle: 1
[*] added channel id: 15 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 16 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 16
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
[*] added process id: 3438, handle: 2
[*] added channel id: 17 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 18 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 18
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
[*] added process id: 3444, handle: 3
[*] added channel id: 19 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 3
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
